src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -13,7 +13,28 @@
   </PropertyGroup>
   
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'net451' OR '$(TargetFramework)' == 'net461' ">
-    <DefineConstants>DESKTOP</DefineConstants>
+    <DefineConstants>$(DefineConstants);DESKTOP</DefineConstants>
+  </PropertyGroup>
+
+  <!-- feature flags -->
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <DefineConstants>$(DefineConstants);HAVE_CREATE_ECDSA;</DefineConstants>
```

Comment:
>HAVE_CREATE_ECDSA [](start = 40, length = 17)

nit: consider SUPPORTS_ECDSA

Reply:
It might be confusing as we do support ECDsa (e.g. ECDsaSecurityKey). We don't support creating of ECDsa using ECParameters in netstandard1.4 i.e. creating an ECDsa object using parameters of a JsonWebKey.

Reply:
then consider calling it SUPPORTS_ECParameters ?

Reply:
I agree, it looks more intuitive. I'll make the change.